tests/session/test_subscription.py

```diff
@@ -69,6 +69,35 @@ async def test_subscribeWithoutContext_subscribesToEventsInAllContexts(
     assert resp["params"]["context"] == context_id
 
 
+@pytest.mark.asyncio
+async def test_subscribeUserContext(websocket, context_id, html):
```

Comment:
it would be nice to have a test for a cusom user context as well. However we can delegate it to WPT tests, up to you

Reply:
I suggest to use WPT, could you please review the first test in WPT https://github.com/web-platform-tests/wpt/pull/50290 ?